### PR TITLE
final fixes

### PR DESCRIPTION
**Pull request by VezmenovDanil**

### CODE REVIEW

File: toDoList/src/app/functional-panel/functional-panel.component.scss [hunk 1]
```diff
@@ -19,12 +19,19 @@
 .input-new-task-style, .create-new-task-button {
     border-radius:10px;
 }
+
 @media (max-width:600px) {
+
     .create-new-task-button{
```
Comment (usatovaDasha) [inline toDoList/src/app/functional-panel/functional-panel.component.scss:25]: check tabs need 2 symbols

File: toDoList/src/app/task-editor/task-editor/task-editor.component.ts [hunk 2]
```diff
@@ -2,35 +2,38 @@ import { Component, Input } from '@angular/core';
...
+
 export class TaskEditorComponent {
+
   @Input() task: any;
   imgEdit: string = "assets/pic/edit.png";
   imgCancel: string = "assets/pic/cancel.png";
 
-  constructor(private workPanelComponent: WorkPanelComponent, private taskService: TaskService, private appComponent: AppComponent) { }
- 
+  constructor(private workPanelComponent: WorkPanelComponent, private taskService: TaskService, private appComponent: AppComponent) { };
```
Comment (usatovaDasha) [inline toDoList/src/app/task-editor/task-editor/task-editor.component.ts:18]: options move to single lines

File: toDoList/src/app/task-item/task-item.component.html [hunk 3]
```diff
@@ -5,8 +5,13 @@
       type="checkbox"
       (click)="taskCheck($event)"
       [checked]="task.isCheck"
+      [attr.disabled]="!deleteButtonStatus ? true : null"
```
Comment (usatovaDasha) [inline toDoList/src/app/task-item/task-item.component.html:8]: [attr.disabled]="!deleteButtonStatus"

File: toDoList/src/app/task.ts [hunk 4]
```diff
@@ -1,3 +1,3 @@
 export class task {
-    constructor(public _id: string, public text: string, public isCheck: boolean) { }
+    constructor(public _id: string, public text: string, public isCheck: boolean) { };
```
Comment (usatovaDasha) [inline toDoList/src/app/task.ts:2]: check tabs

File: toDoList/src/app/tasks.service.ts [hunk 5]
```diff
@@ -8,25 +8,25 @@ export class TaskService {
 
         getTasks(): Observable<any> {   //create Model Here bruh
```
Comment (usatovaDasha) [inline toDoList/src/app/tasks.service.ts:9]: check tabs

File: toDoList/src/app/task-item/task-item.component.html (hunk 3 above)
Comment (VezmenovDanil) [reply toDoList/src/app/task-item/task-item.component.html:8]: it doesn't work

**Pull request merged**